pkg/services/datasources/service/datasource_test.go

```diff
@@ -537,6 +537,74 @@ func TestService_GetHttpTransport(t *testing.T) {
 		require.NotNil(t, configuredOpts.SigV4)
 		require.Equal(t, "es", configuredOpts.SigV4.Service)
 	})
+
+	t.Run("Should use 'es' if OpenSearch and not serverless configured in JsonData", func(t *testing.T) {
```

Comment:
I think we can collapse these two tests into one / only initialize the services once since the only difference is the JSON field value.

Alternatively, since it's not even worth firing up the DB when all you really need to test is `func awsServiceNamespace` - you could just add a small unit test for that instead 😄

Reply:
Thanks @wbrowne! Yea, should have done that directly. Changed to test `awsServiceNamespace` 😊